cmd/ddev/cmd/describe.go

```diff
@@ -180,7 +181,7 @@ func renderAppDescribe(app *ddevapp.DdevApp, desc map[string]interface{}) (strin
 			if _, ok := desc["mailpit_https_url"]; ok {
 				mailpitURL = desc["mailpit_https_url"].(string)
 			}
-			t.AppendRow(table.Row{"Mailpit", "", fmt.Sprintf("Mailpit: %s\n`ddev launch -m`", mailpitURL)})
+			t.AppendRow(table.Row{"Mailpit", "", fmt.Sprintf("Mailpit: %s\n`ddev launch -m` or `ddev mailpit`", mailpitURL)})
```

Comment:
I would be in favor of just saying `ddev mailpit`

Reply:
Done